core/src/main/kotlin/org/evomaster/core/output/service/naming/NamingStrategy.kt

```diff
@@ -0,0 +1,9 @@
+package org.evomaster.core.output.service.naming
```

Comment:
`.service.` packages have a special meaning, altough it is not enforced in the build. see `docs/for_developers.md`. rather use for example `org.evomaster.core.output.naming`

Reply:
Done